### PR TITLE
Entity lecture2 linq

### PR DESCRIPTION
**Pull request by wi9916**

### CODE REVIEW

File: Entity/Entity/Program.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace Entity
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var SameShop = new VirtualShop();
+
+            //    SameShop.GetAllProduct();
```
Comment (yevhenbilushko) [inline Entity/Entity/Program.cs]: please remove this comments

File: Entity/Entity/Services/VirtualShop.cs [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 2, ProductId = 2 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 2, ProductId = 6 });
+            MyProductsFromCompanies.Add(new ProductsFromCompany { CompanyId = 3, ProductId = 7 });
+        }
+        public void GetAllProduct()
+        {
+            Speaker.Output("===========================================================================");
+            Speaker.Output("All Product");
+
+            //var products = from e in MyProduct
```
Comment (yevhenbilushko) [inline Entity/Entity/Services/VirtualShop.cs]: yeah this is on of the solution
but either uncomment it and move to separate method or remove this commented code

File: Entity/Entity/Services/VirtualShop.cs [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+            //               orderby e.Name
+            //               select e;
+
+            var products = MyProducts.Select(e => e).OrderBy(e => e.Name);
+
+            foreach (var p in products)
+            {
+                Speaker.Output("Name - " + p.Name);
```
Comment (yevhenbilushko) [inline Entity/Entity/Services/VirtualShop.cs:62]: try to use string interpolation

Comment (wi9916) [reply Entity/Entity/Services/VirtualShop.cs:62]: Use Console.WriteLine($"Name - {p.Name} +  Count - {p.Count}"); ?

Comment (wi9916) [reply Entity/Entity/Services/VirtualShop.cs:62]: So i must not use my class Speaker?

**Review by yevhenbilushko — APPROVED** [review]

**Pull request closed without merging**